scrapers_next/ar/people.py

```diff
@@ -131,3 +132,137 @@ def process_item(self, item):
         p = PartialMember(name=name, chamber=chamber, url=self.source.url)
 
         return LegDetail(p, source=source)
+
+
+class SenDetail(HtmlPage):
+    example_source = (
+        "https://senate.arkansas.gov/senators/missy-irvin/"
+    )
+
+    def process_page(self):
+
+      #  table_div = CSS(".col-md-8").match(self.root)[0]
+
+        heading = CSS(".heading").match(self.root)[0]
+        name = re.sub(r"Senator ", "", heading.text_content()).strip()
+
+        rows = XPath("//div[contains(@class,col-md-8)]/ul/li").match(self.root[0]) # CSS(".row").match(table_div)
+
+        
+        image_container = CSS(".col-md-4").match(self.root)[0]
+        image = XPath("//img[contains(@alt, '" + name + "')]").match(image_container)[0].get('src')
+        image = image.split("?")[0]
+
+        biocontainer = CSS(".panel-body").match(self.root)[0]
+        bio = XPath("//a/@href").match(biocontainer)[0]
+        committees = XPath("//div[contains(@class,base-text)]/ul/li").match(self.root)
+        
+        
+
+        table = {
+            "Party": "",
+            "Phone": "",
+            "Email": "",
+            "District": "",
+            "Seniority": "",
+            "Occupation": "",
+            "Party": "",
+            "District Address": "",
+            "Church Affiliation:": "",
+            "Veteran": "",
+            "Public Service": "",
+            "Legislative Service": "",
+            "Biography": "",
+        }
+
+        for row in rows:
+            try: 
+                text = row.text_content()
+                text = text.split(": ")
+                if (len(text) < 2):
+                    continue
+                name = text[0]
```

Comment:
This use of `name` is overwriting the actual name of the Senator since it is past when `p` is created